nio-core/src/main/scala/zio/nio/core/channels/DatagramChannel.scala

```diff
@@ -1,40 +1,145 @@
 package zio.nio.core.channels
 
 import java.io.IOException
+import java.net.{ DatagramSocket => JDatagramSocket, SocketAddress => JSocketAddress }
 import java.nio.channels.{ DatagramChannel => JDatagramChannel }
-import java.net.{ DatagramSocket => JDatagramSocket }
 
-import zio.nio.core.{ ByteBuffer, SocketAddress }
 import zio.{ IO, UIO }
+import zio.nio.core.{ ByteBuffer, SocketAddress, SocketOption }
 
+/**
+ * A [[java.nio.channels.DatagramChannel]] wrapper allowing for basic [[zio.ZIO]] interoperability.
+ */
 final class DatagramChannel private[channels] (override protected[channels] val channel: JDatagramChannel)
     extends GatheringByteChannel
     with ScatteringByteChannel {
 
-  def bind(local: SocketAddress): IO[IOException, DatagramChannel] =
-    IO.effect(new DatagramChannel(channel.bind(local.jSocketAddress))).refineToOrDie[IOException]
+  /**
+   * Binds this channel's underlying socket to the given local address. Passing `None` binds to an
+   * automatically assigned local address.
+   *
+   * @param local the local address
```

Comment:
it would be good to mentioned if this is `127.0.0.1` or `0.0.0.0`

Reply:
I think this is `127.0.0.1` but I don't know how to verify this exactly.